cmd/internal/ecs/ecs.go

```diff
@@ -64,6 +69,20 @@ var RemoveFromECSCmd = &cobra.Command{
 	Hidden: true,
 }
 
+var PrintCloudFormationFragmentCmd = &cobra.Command{
+	Use:   "cf-fragment",
+	Short: "Print an AWS CloudFormation fragment for adding the Postman Insights Agent to AWS ECS.",
+	Long:  "Print a code fragment that can be inserted into a CloudFormation template to add the Postman Insights Agent as a sidecar container.",
+	RunE:  printCloudFormationFragment,
+}
+
+var PrintECSTaskDefinitionCmd = &cobra.Command{
+	Use:   "task-def",
+	Short: "Print an AWS ECS task definition for running the Postman Insights Agent in daemon mode.",
+	Long:  "Print a task definition that can be added to an ECS cluster to run the Postman Insights Agent as a daemon in host-networking mode on every EC2 instance in the cluster.",
+	RunE:  printECSTaskDefinition,
```

Comment:
I wonder whether these subcommands should be set up to call telemetry.Init(false) to disable logging from the telemetry system -- do they otherwise log?  I think for the K8S injector we tried to make sure it did not, but the use case is not quite the same.

Reply:
The three lines below are the only things extra that is printed. Only the first two lines are from the telemetry system, and won't be there for the release build and are to stderr. I've modified the third to be printed to stderr. Everything else is on stdout.
```
[INFO] Telemetry unavailable; no Amplitude key configured.
[INFO] This is caused by building from source rather than using an official build.
[INFO] Postman Insights Agent 0.0.0
```